libsolidity/interface/StandardCompiler.cpp

```diff
@@ -225,6 +228,50 @@ Json::Value collectEVMObject(eth::LinkerObject const& _object, string const* _so
 	return output;
 }
 
+boost::optional<Json::Value> checkKeys(Json::Value const& _input, set<string> const& _keys)
+{
```

Comment:
I would also check at this point whether `_input` is an object

Reply:
but only error out when it also exists and is not an object ;)

Reply:
Actually, I can just add in on my PR which is based on this one

Reply:
Ok!